.github/workflows/testing.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         if echo $logs | grep -i "exited with code"; then
           echo "Found errors in Docker logs"
           exit 1
-        fi
+        fi
```

Comment:
nit: better to end the file with one empty line
[PEP8 "No newline at end of file"](https://stackoverflow.com/questions/71976465/pep8-no-newline-at-end-of-file)